README.md

```diff
@@ -79,6 +79,26 @@ public io.opentracing.Tracer tracer() {
 }
 ```
 
+### Properties
```

Comment:
These are missing:
`opentracing.spring.cloud.redis.enabled` - Enable Redis tracing.
`opentracing.spring.cloud.jdbc.withActiveSpanOnly` - Only trace JDBC calls if they are part of an active Span.
`opentracing.spring.cloud.jdbc.ignoreStatements` - Set of JDBC statements to not trace.

Reply:
Thank you! It is done!